README.md

```diff
@@ -281,15 +295,15 @@ If you do not have root access pyodbc will not work if you use brew. Using the
 docker compose you can set up the full stack and it will also set the correct
 environment variables.
 
-To run the database dependencies used by Crawler and also Lighthouse, there is a
-separate configuration for Docker Compose. This is shared with Lighthouse so if
-you start these dependencies here, there's no need to also attempt to do so in
-the Lighthouse repository. They are the same resources in both and the second
-one to be started will show exceptions about ports already being allocated:
+To run the dependencies used by Crawler and also Lighthouse, there is a separate
+configuration for Docker Compose. This is shared with Lighthouse so if you start
+these dependencies here, there's no need to also attempt to do so in the
+Lighthouse repository. They are the same resources in both and the second one to
+be started will show exceptions about ports already being allocated:
```

Comment:
This is a possible problem that people could have. Maybe we could mark it as a **Note**? so is not part of the procedure, but something they should know if they have this issue.

Reply:
This text has been here for months and is reciprocated in the other repositories as well, but I'll change it here if you think it'll help.